src/js/core/util/deferredWhen.js

```diff
@@ -61,6 +61,9 @@
 					 * @member ns.util.deferredWhen
 					 * @private
 					 */
+					progressValues,
```

Comment:
These variables should be moved above description of "updateFunc" ?

Reply:
done